index.js

```diff
@@ -131,33 +131,46 @@ function linkifyAndAdd(parent, message) {
 }
 
 function addItem({era, when, user, message, channel}, showUserLink) {
-    const when_str = toDateString(when)
-
     const item = DOM.create("li");
-    if (channel.startsWith("#")) {
-        let link = urls.logbot(channel.substr(1), when, user);
-        const ahref = DOM.create("a", {href: link});
-        ahref.textContent = when_str;
-        item.appendChild(ahref);
-    } else {
-        const text = DOM.createText(`${when_str}`);
-        item.appendChild(text);
-    }
 
-    // Let's try to find bug numbers.
-    item.appendChild(DOM.createText(" - "));
+    const header = DOM.create("div", {class: "update-header"});
+
+    const name = DOM.create("strong", {class: "name"});
+    const nameText = DOM.createText(user);
     if (showUserLink) {
         const link = DOM.create("a", {href: urls.user_page(user) });
-        link.textContent = user;
-        item.appendChild(link);
-        item.appendChild(DOM.createText(" - "));
+        link.appendChild(nameText);
+        name.appendChild(link);
+    } else {
+        name.appendChild(nameText);
+    }
+    header.appendChild(name);
+
+    header.appendChild(DOM.createText(" "));
+
+    const time = DOM.create("small", {class: "time"});
+    let timeText = DOM.createText(toDateString(when));
+    if (channel.startsWith("#")) {
+        let href = urls.logbot(channel.substr(1), when, user);
+        const link = DOM.create("a", {href});
+        link.appendChild(timeText);
+        time.appendChild(link);
+    } else {
+        time.appendChild(timeText);
     }
-    linkifyAndAdd(item, message);
+    header.appendChild(time);
+
+    header.appendChild(DOM.createText(" "));
+
+    const edit = DOM.create("a", {class: "edit", href: urls.edit(user, era), target: "_blank"});
+    edit.appendChild(DOM.create("img", {src: "icons/edit.png", height: 10, alt: "edit"}));
```

Comment:
nit: use `em` units instead of `10` for the height, maybe?

Reply:
I'm not sure if it's appropriate to specify raster image size with `em`,
it will result in always applying scaling, and not-clear rendering result, especially for small image like this.

I'd like to leave it to other issue.
maybe we could add vector icon font or SVG icon instead (I thought SVG is supported by most modern browsers).

Reply:
Filed https://github.com/mrgiggles/histoire/issues/14

Reply:
Oh yes, you're right, sorry for the noise!